lib/providers/portion/PortionProvider.ts

```diff
@@ -39,6 +38,33 @@ export class PortionProvider implements IPortionProvider {
     return tokenOutCurrencyAmount.multiply(new Fraction(portion.bips, 10000));
   }
 
+  getPortionQuoteAmount(
+    portionAmountToken: CurrencyAmount<Currency>,
+    quoteCurrencyAmount: CurrencyAmount<Currency>,
+    amount: CurrencyAmount<Currency>
```

Comment:
What amount is this referring to? looks like it's the amount of the exact out the swapper requested. Could we make that more clear in var name.